packages/dma-contracts/tasks/service-registry/index.ts

```diff
@@ -104,59 +166,16 @@ async function pushConfigToRemote(configPath: string, network: string, ethers: a
   console.log(`ServiceRegistry: ${serviceRegistry.address}`)
   console.log('==============================\n')
 
-  for (const serviceName of Object.keys(property)) {
-    const service = property[serviceName] as ConfigEntry
+  if (property.name !== undefined) {
+    // Single entry
+    await pushEntryToRemote(serviceRegistry, property as ConfigEntry)
+  } else {
+    // Multiple entries
+    for (const serviceName of Object.keys(property)) {
+      const service = property[serviceName] as ConfigEntry
 
-    if (!service.serviceRegistryName) {
-      console.log(`[✅][LOCAL CONFIG] ${service.name}: ${service.address}`)
-      continue
+      await pushEntryToRemote(serviceRegistry, service)
     }
```

Comment:
cant comment below, but maybe this could be refactored to subtasks ?

```ts
   if (taskArgs.pushconfig) {
      await pushConfigToRemote(taskArgs.pushconfig, network, ethers)
    } else if (taskArgs.showlocal) {
      showLocalConfig(taskArgs.showlocal, network)
    } else if (taskArgs.showremote) {
      await showRemoteConfig(taskArgs.showremote, network, ethers)
    } else {
      throw new Error('Either --pushconfig, --showlocal or --showremote must be specified')
    }
```